azurerm_notification_hub/main.tf

```diff
@@ -0,0 +1,62 @@
+provider "azurerm" {
+  version = "~>1.43"
+}
+
+terraform {
+  backend "azurerm" {}
+}
+
+data "azurerm_key_vault" "key_vault" {
+  name                = var.key_vault_name
+  resource_group_name = var.resource_group_name
+}
+data "azurerm_key_vault_secret" "notification_hub_gcm_key" {
+  name         = "notification-hub-01-gc-m-key"
```

Comment:
it is correct this value "notification-hub-01-gc-m-key" is assigned through a variable?

Reply:
Yes is correct, because this values, like the others, are statics and must be manually inserted in the azure keyvault